docs/ref/contrib/auth.txt

```diff
@@ -560,6 +568,17 @@ The following backends are available in :mod:`django.contrib.auth.backends`:
         Returns whether the ``user_obj`` has any permissions on the app
         ``app_label``.
 
+    .. method:: with_perm(perm, obj=None)
+
+        .. versionadded:: 1.11
+
+        Returns all users who have the permission ``perm``. Returns an empty
+        set if :attr:`~:attr:`~django.contrib.auth.models.CustomUser.is_active``
```

Comment:
This seems like it should be saying that it doesn't return inactive users but is saying something else.